Bio/Graphics/GenomeDiagram/_GraphSet.py

```diff
@@ -155,21 +155,3 @@ def __str__(self):
         outstr.append("%d graphs" % len(self._graphs))
         outstr = "\n".join(outstr)
         return outstr
-
-
-################################################################################
-# RUN AS SCRIPT
-################################################################################
-
-if __name__ == '__main__':
-
-    # Test code
-    gdgs = GraphSet(0, 'test data')
-
-    testdata1 = [(1, 10), (5, 15), (10, 20), (20, 40)]
-    testdata2 = [(250, .34), (251, .7), (252, .7), (253, .54), (254, .65)]
-
-    gdgs.add_graph(testdata1, 'TestData 1')
```

Comment:
I see no use of ``. add_graph`` in ``Tests/test_GenomeDiagram.py`` so this is not covered.

Reply:
There is no 'add_graph' in _GraphSet.py, I think is superseded with new_graph.

Reply:
Oh, so this old self-test had stopped working too. We can clarify this in the merge commit comment.